test/conf/cassandra-murmur.yaml

```diff
@@ -39,6 +39,6 @@ compaction_throughput: 0MiB/s
 row_cache_class_name: org.apache.cassandra.cache.OHCProvider
 row_cache_size: 16MiB
 user_defined_functions_enabled: true
-scripted_user_defined_functions_enabled: true
+scripted_user_defined_functions_enabled: false
```

Comment:
should we just delete all these lines from test cassandra*.yaml files?
(since the deprecated default is `false` )

Reply:
We can but I wasn't completely sure how things will go in the other ticket as this is only deprecated now in Config class. (And in general there is this tendancy to leave config in the Config class deprecated, maybe even forever? I hear different opinions)
Also, wanted to be sure during testing we don't get sabotaged getting the value from somewhere else that I miss, so flipping at first to be sure they are disabled and fail sounded as a good first choice

Reply:
ok to leave. i trust your judgement.

Reply:
Thanks, I will put a note on the other ticket that depending on how it goes we might want to remove them (we should if there are no more reasons to keep them, decluttering is important)